src/realm/util/interprocess_condvar.cpp

```diff
@@ -190,8 +241,10 @@ void InterprocessCondVar::init_shared_part(SharedPart& shared_part)
 
 void InterprocessCondVar::release_shared_part()
 {
-#ifdef REALM_CONDVAR_EMULATION
+#ifdef REALM_CONDVAR_EMULATION 
+#ifndef _WIN32
     File::try_remove(m_resource_path);
+#endif
 #else
 #endif
```

Comment:
> It has the same ifdef-structure

In multiple places, just in this file, the structure is simply:
```C++
#ifndef _WIN32
...
#endif
```
However, if we _really_ want to fix this, we should go for always-defined platform identifiers. Have all platforms covered in the cases and have an `#error` in `#else`. That is the most safe approach as it forces us to go through all cases and make a conscious decision.